PowerUp/app/app.iml

```diff
@@ -86,6 +86,7 @@
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/rs" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/symbols" />
       <excludeFolder url="file://$MODULE_DIR$/build/outputs" />
+      <excludeFolder url="file://$MODULE_DIR$/build/tmp" />
```

Comment:
I don't know why it was made, maybe it happened when I tried to directly
commit from Android Studio. I tried it out for the first time. I always
used Git Bash to separately add files. Will undo everything very soon.

On Wed, Nov 4, 2015 at 2:56 PM, Alma Castillo Antolin <
notifications@github.com> wrote:

> In PowerUp/app/app.iml
> <https://github.com/systers/powerup-android/pull/28#discussion_r43857713>:
>
> > @@ -86,6 +86,7 @@
> >        <excludeFolder url="file://$MODULE_DIR$/build/intermediates/rs" />
> >        <excludeFolder url="file://$MODULE_DIR$/build/intermediates/symbols" />
> >        <excludeFolder url="file://$MODULE_DIR$/build/outputs" />
> > +      <excludeFolder url="file://$MODULE_DIR$/build/tmp" />
>
> Same: should not submit this. Why was this change made?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/powerup-android/pull/28/files#r43857713>.
>